### PR TITLE
remove legacy command

### PR DESCRIPTION
**Pull request by winni2k**
See #1120

### CODE REVIEW

**Comment by jeromekelleher** [comment]
Thanks @winni2k , this is great. Just FYI I've not gone through and reviewed as you've marked it as a draft.

**Comment by winni2k** [comment]
Yup, I still need to move the Macs and newick commands.

**Comment by winni2k** [comment]
The PR for macs and newick in tskit is at https://github.com/tskit-dev/tskit/pull/772

**Comment by jeromekelleher** [comment]
This is great, thanks @winni2k! I think we just need to add the list of commands removed from the msp cli to the "breaking changes" section of the CHANGELOG and we're good to go.

File: CHANGELOG.rst [hunk 1]
```diff
@@ -35,6 +35,10 @@
 - The ``simulate`` function only takes one positional argument, and all other
   arguments are keyword-only.
 
+- The ``msp`` CLI has been stripped of all sub-commands except for
+  ``simulate`` and ``mutate``. All removed sub-commands except for ``newick``
```
Comment (jeromekelleher) [inline CHANGELOG.rst:39]: This isn't quite right I think. How about "The functionality of these old sub-commands is now provided by tskit, either through the ``tskit`` CLI or API functions.".

**Review by jeromekelleher — COMMENTED** [review]
Great, thanks @winni2k. One minor change and we're good to merge.

**Review by jeromekelleher — APPROVED** [review]

**Comment by jeromekelleher** [comment]
Woot, that's 600 lines of code we don't need to carry around any more, thanks @winni2k !

**Comment by winni2k** [comment]
I love deleting code :D

**Pull request merged**